docs/signed-apk-android.md

```diff
@@ -29,7 +29,7 @@ It will output the directory of the JDK, which will look something like this:
 
     /Library/Java/JavaVirtualMachines/jdkX.X.X_XXX.jdk/Contents/Home
 
-Navigate to that directory by using the command `cd /your/jdk/path` and use the keytool command with sudo permission as shown below.
+Navigate to that directory by using the command `cd /your/jdk/path` and use the keytool command with sudo permission as shown below. Remember to edit `my-upload-key` and `my-key-alias`.
```

Comment:
```suggestion
Navigate to that directory by using the command `cd /your/jdk/path` and use the keytool command with sudo permission as shown below. It then generates the keystore as a file called `my-upload-key.keystore`.
```

Reply:
I believe this does not clarify the person needs to edit the placeholders with their own value.

Reply:
But this is more consistent with the Windows paragraph just above. If you wish to edit them, you need to update them both